addons/hearing/CfgWeapons.hpp

```diff
@@ -13,42 +13,42 @@ class CfgWeapons {
         };
     };
 
+    #define HEARING_PROTECTION_VICCREW EGVAR(hearing,protection) = 0.85; EGVAR(hearing,lowerVolume) = 0.6;
+    #define HEARING_PROTECTION_EARMUFF EGVAR(hearing,protection) = 0.75; EGVAR(hearing,lowerVolume) = 0.5;
+    #define HEARING_PROTECTION_PELTOR EGVAR(hearing,protection) = 0.75; EGVAR(hearing,lowerVolume) = 0.2;
```

Comment:
I'd reduce the `lowerVolume` value to `0.0` because of the sound enhancing features / microphones